kong/clustering/data_plane.lua

```diff
@@ -252,11 +252,11 @@ function _M:communicate(premature)
 
           if msg.type == "reconfigure" then
             if msg.timestamp then
-              ngx_log(ngx_DEBUG, _log_prefix, "received reconfigure frame from control plane with timestamp: ",
+              ngx_log(ngx_INFO, _log_prefix, "received reconfigure frame from control plane with timestamp: ",
```

Comment:
@lukego 

ngx_INFO needs to be localized here:
https://github.com/Kong/kong/blob/46bcae0b13c9c9e4d5181e0bfcd2aa2a3c93b6a4/kong/clustering/data_plane.lua#L30

Reply:
After rebasing, the problem is fixed.